deploy/charts/emqx-operator/templates/webhook-validating-configuration.yaml

```diff
@@ -90,24 +90,3 @@ webhooks:
     resources:
     - emqxes
   sideEffects: None
-- admissionReviewVersions:
-  - v1
-  - v1beta1
-  clientConfig:
-    service:
-      name: {{ include "emqx-operator.fullname" . }}-webhook-service
-      namespace: {{ .Release.Namespace }}
-      path: /validate-apps-emqx-io-v1beta4-rebalance
-  failurePolicy: Fail
-  name: validator.rebalance.emqx.io
-  rules:
-  - apiGroups:
-    - apps.emqx.io
-    apiVersions:
-    - v1beta4
-    operations:
-    - CREATE
-    - UPDATE
-    resources:
-    - rebalances
-  sideEffects: None
```

Comment:
Why delete this ?

Reply:
Will add it back when coding controller，otherwise it will cause ci to fail